content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -161,8 +161,8 @@ The [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/cu
 API resource allows you to define custom resources.
 Defining a CRD object creates a new custom resource with a name and schema that you specify.
 The Kubernetes API serves and handles the storage of your custom resource.
-The name of a CRD object must be a valid
-[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+The name of the CRD object itself must be a valid
+[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) derived from the defined resource name and its API group; see [how to create a CRD](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions#create-a-customresourcedefinition) for more details.
```

Comment:
I guess it makes sense to mention CRs here as well as they are what really matter and surprise users.

Reply:
OK, I have force-pushed a revision that restores a statement about that here.

Reply:
Thanks, lgtm